packages/manager/modules/pci/src/projects/project/training/jobs/info/info.html

```diff
@@ -33,58 +33,84 @@
         <div class="col-md-4">
             <oui-tile
                 class="h-100"
-                data-heading="{{:: 'pci_projects_project_training_jobs_info_title_1' | translate  }}"
+                data-heading="{{   'pci_projects_project_training_jobs_info_title_1' | translate  }}"
             >
                 <oui-tile-definition
-                    data-term="{{:: 'pci_projects_project_training_jobs_id' | translate  }}"
+                    data-term="{{   'pci_projects_project_training_jobs_name' | translate  }}"
                 >
                     <oui-tile-description
-                        data-ng-bind="$ctrl.job.id"
+                        data-ng-bind="$ctrl.job.spec.name"
                     ></oui-tile-description>
                 </oui-tile-definition>
                 <oui-tile-definition
-                    data-term="{{:: 'pci_projects_project_training_jobs_region' | translate  }}"
+                    data-term="{{   'pci_projects_project_training_jobs_region' | translate  }}"
                 >
                     <oui-tile-description
-                        data-ng-bind="$ctrl.job.region"
+                        data-ng-bind="$ctrl.job.spec.region"
                     ></oui-tile-description>
                 </oui-tile-definition>
                 <oui-tile-definition
-                    data-term="{{:: 'pci_projects_project_training_jobs_creator' | translate  }}"
+                    data-term="{{   'pci_projects_project_training_jobs_created' | translate  }}"
                 >
                     <oui-tile-description
-                        data-ng-bind="$ctrl.job.user"
+                        data-ng-bind="$ctrl.job.createdAt | date:'medium'"
                     ></oui-tile-description>
                 </oui-tile-definition>
+
                 <oui-tile-definition
-                    data-term="{{:: 'pci_projects_project_training_jobs_created' | translate  }}"
+                    data-term="{{   'pci_projects_project_training_jobs_info_title_5' | translate  }}"
+                    data-ng-if="$ctrl.job.status.history"
                 >
-                    <oui-tile-description
-                        data-ng-bind="$ctrl.job.created | date:'medium'"
-                    ></oui-tile-description>
+                    <oui-tile-description>
+                        <div
+                            class="oui-progress-tracker oui-progress-tracker_vertical mt-3"
+                        >
+                            <ol class="oui-progress-tracker__steps">
+                                <li
+                                    ng-repeat="status in $ctrl.job.status.history"
```

Comment:
It is considered as a **best practice** to use `track by`.
Could you please add it if possible?